features/step_definitions/operators.rb

```diff
@@ -160,3 +160,44 @@
   cb_name = 'operator_channel_name' unless cb_name
   cb[cb_name] = cluster_version('version').channel.split('-')[1]
 end
+
+Given /^I wait(?: up to (\d+) seconds)? for operator "([^"]*)" to become conditions: "([^"]*)"$/ do | time, operator, conditions |
```

Comment:
Lets make the conditions a table for brevity.

```
Given /^I wait(?: up to #{NUMBER} seconds)? for operator #{QUOTED} to meet conditions:$/ do | time, operator, conditions |
  arr_conditions = conditions.raw.flatten
  ...
```

Reply:
Hi, Aleks, I prefer current step def, one single line is more of brevity than additional table lines.
If you think the single line is a little lengthy, I suggest Ke remove characters `conditions: ` and revise it to be:
Given /^I wait(?: up to (\d+) seconds)? for operator "([^"]*)" to become "([^"]*)"$/ do | time, operator, conditions |

The frequent uses we actually possibly need would be AFAIK:
Given I wait up to XXX seconds for operator "XXX" to become "True False False"
Given I wait up to XXX seconds for operator "XXX" to become "Progressing=True"
Sufficient brevity.

Reply:
`"True False False"`, this is hard to read/review/maintain.
We should force to use format `Progressing=True`

Reply:
oc get co shows Available Progressing Degraded order "True False False", this is a special but very common meaning to everyone, therefor Ke specially implemented for this. So not hard to read. Let's not make the special implementation effort wasted.

Reply:
For those people who are familiar with status of operator, uses 'True False ..." is simple and convenient, for those who are unfamiliar, there is another option: "Progressing=True".

Reply:
If we implement such option, we should make sure in code that proper order of checking will be performed. We should not rely on order provided by API. Even better, we can have 
`operator "..." becomes available/non-progressing/non-degraded within ## seconds`
or
`operator "..." becomes non-available/progressing/degraded within ## seconds`

Should be readable by everybody. The reason to use Gherkin is to make tests readable by non-technical leads and managers (whether they will ever do, I don't know). They will certainly have troubles reading `true false false`.

Reply:
I changed as following:
```
Given /^I wait(?: up to (\d+) seconds)? for operator "([^"]*)" to become conditions[available/non-progressing/non-degraded]: "([^"]*)"$/ do | time, operator, conditions |
```
the conditions `[available/non-progressing/non-degraded]:` is optional.